tico/quantization/algorithm/gptq/gptq.py

```diff
@@ -90,6 +88,31 @@ def add_batch(self, inp, out):
             )  # inp.shape == (k_h*k_w*in_channels / groups, batch * groups * flattened_patches)
             # so inp.matmul(inp.t()).shape == (k_x*k_y*in_channels / groups, k_x*k_y*in_channels / groups) == W.flatten(1)
 
+        if isinstance(self.layer, nn.Conv1d):
+            # nn.Conv1d is basically the same as nn.Conv2d so we can use the same idea as for nn.Conv2d
+            # TODO reduce code duplication
+            # represent conv1d as conv2d(1, k) on reshpaed_input(batch, in_channels, 1, L)
+            unfold = nn.Unfold(
+                (1, self.layer.kernel_size[0]),
+                dilation=(1, self.layer.dilation[0]),
+                padding=(1, self.layer.padding[0]),
```

Comment:
Maybe? Seems there's no difference to the result though.

```suggestion
                padding=(0, self.layer.padding[0]),
```